lib/symboldatabase.cpp

```diff
@@ -1981,8 +1981,9 @@ void Variable::evaluate(const Settings* settings)
 
     const Library * const lib = &settings->library;
 
+    bool isContainer = false;
     if (mNameToken)
-        setFlag(fIsArray, arrayDimensions(settings));
+        setFlag(fIsArray, arrayDimensions(settings, isContainer));
```

Comment:
for output parameters I'd rather pass by address. the `&` is an indication it's an output parameter.